---
title: "Make a data publication plan"
teaching: 5
exercises: 30-60
questions:
- "How do I create a data publication plan?"
- ""
- ""
objectives:
- ""
- ""
keypoints:
- "Plan ahead - by knowing where the data will be published already during the project, the data and metadata can be formatted according to the repository's requirements, thus reducing the labour when submitting"
- "" 
- ""
- ""
---
<!-- 
Present the exercise, purpose, how
Present the 3 scenarios
Present expected outcome, how to create the plan
Present solutions
-->
## Exercise - make a data publication plan
During this exercise you will create a plan for making research data available to the public, by identifying a suitable repository and finding out what the requirements are for submission. Three data output scenarios will be presented, with different data types.

### Data publication plan
Things to consider when making a data publication plan:
* What types of outputs will you be creating or collecting?
* What are suitable repositories for your outputs?
* What are the documentation guidelines for the repositories?
* What preparations are needed to make data and documentation (metadata) ready for submission?
* Under what licenses will your research outputs be shared?

### The scenarios
Select one of the scenarios below and answer the questions under the Steps section.
<!-- for each project, present a brief description of the research question and the main outcomes; data types, excel sheets, analysis scripts? -->
> #### Project A <!-- ArrayExpress -->
> 
{: .solution}

> #### Project B <!-- PRIDE -->
> 
{: .solution}

> #### Project C <!-- SLL Figshare -->
> 
{: .solution}

### Steps

#### What types of outputs will you be creating or collecting?
Make a list of the outputs, start with the main, most important, data type(s) for the study

* `<data type>`
* `<data type>`

#### What are suitable repositories for your outputs?
Depositing your data in a publicly accessible recognised repository which assigns a globally persistent identifier (e.g. a DOI) ensures that your dataset continues to be available to both humans and machines in a usable form in the future.
Try to identify suitable repositories for your main data type(s).

##### Tools
* [ELIXIR Deposition Databases for Biomolecular Data](https://www.elixir-europe.org/platforms/data/elixir-deposition-databases)
* [EBI Repository Wizard](https://www.ebi.ac.uk/submission/)
* [Scientific Data Repository Guidance](https://www.nature.com/sdata/policies/repositories#life)

##### Repositories
* `<data type>: <repository>, <link>`


#### What are the documentation guidelines for the repositories?
Repositories will often have documentation guidelines that are according to domain-relevant standards, which will improve the FAIRness of your data. Organising your output documentation according to these guidelines and standards from the start will make your FAIRification journey much easier.

List “documentation entities” that the repository will expect you to provide

Links `<repository>`

* Study/project description : `<link to information at repository>`
* Sample descriptions: `<link to information at repository>`
* `<other>` descriptions: `<link to information at repository>`

#### What preparations are needed to make data and documentation (metadata) ready for submission?
What is needed in order to be able to do a submission to the chosen repository? Do you e.g. have all metadata at hand, is the (meta) data in an appropriate format?
<!-- not sure what we aim for in this section -->

#### Under what licenses will your research outputs be shared?
Does the repository decide the license, or is this decided by you as a submitter? Identify the terms of the repoccitory. If you decide, which license would you choose and why?
<!-- is this accurate? -->

### Solutions
  > #### Solution project A <!-- ArrayExpress -->
  > ##### Data types
  > * RNAseq
  > * Gene expression
  >
  > ##### Repository 
  > * Data type RNAseq: [ArrayExpress](https://www.ebi.ac.uk/arrayexpress/)
  > * Data type gene expession: [ArrayExpress](https://www.ebi.ac.uk/arrayexpress/)
  >
  > ##### Documentation guidelines
  > * [Submission help](https://www.ebi.ac.uk/arrayexpress/help/submissions_overview.html) 
  >
  > ##### Needed preparations
  > * Collect the sample descriptions in txt format according to the metadata standard [Minimum Information About a Sequencing Experiment (MINSEQE)](http://fged.org/projects/minseqe/)
  > * Collect the sequence library descriptions according to [https://www.ebi.ac.uk/fg/annotare/help/seq_lib_spec.html](https://www.ebi.ac.uk/fg/annotare/help/seq_lib_spec.html)
  > * Make sure that sequence files are in either fastq or bam format, and compressed with gzip
  >
  > ##### License
  > Unless stated otherwise Creative Commons No rights reserved (CC0) is applied to all EBI resources, see further [https://www.ebi.ac.uk/licencing](https://www.ebi.ac.uk/licencing)
  >
  {: .solution}

  > #### Solution project B <!-- PRIDE -->
  > ##### Data types
  > ##### Repository 
  > ##### Documentation guidelines
  > ##### Needed preparations
  > ##### License
  > Unless stated otherwise Creative Commons No rights reserved (CC0) is applied to all EBI resources, see further [https://www.ebi.ac.uk/licencing](https://www.ebi.ac.uk/licencing)
  > 
  {: .solution}

  > #### Solution project C <!-- SLL Figshare -->
  > ##### Data types
  > ##### Repository 
  > ##### Documentation guidelines
  > ##### Needed preparations
  > ##### License
  > 
  {: .solution}

